packages/renderer/src/lib/welcome/ProviderNotInstalled.svelte

```diff
@@ -34,7 +34,14 @@ let preflightChecks: CheckStatus[] = [];
         <div class="flex flex-col">
           <p class="mb-4 items-center list-inside">{detectionCheck.status ? '✅' : '❌'} {detectionCheck.name}</p>
           {#if detectionCheck.details}
-            Details: <p class="text-gray-300 w-full break-all">{detectionCheck.details}</p>
+            <p class="text-gray-300 w-full break-all">Details:</p>
+            <ul class="list-none">
+              {#each detectionCheck.details.split(';') as detail}
```

Comment:
That is Windows specific

Reply:
So changes not required ?

Reply:
Yeah, I don't believe this would be required / shouldn't be added as it's Windows specific. It won't occur on macOS / Linux.